modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -75,10 +78,49 @@ const Diagram = ({
 		radius: newPinSettings.defaultRadius,
 	});
 
+	const importPinSchema = () => {
+		const textDatas = [];
+		const pinDatas = [];
+		const parser = new DOMParser();
+		const xmlImage = parser.parseFromString(svgString, 'image/svg+xml');
+		const rootLevel = xmlImage.getElementById('Livello_Testi');
```

Comment:
1. Yes, at this time we have this info only from api. In future improvements we must include language choice

Reply:
We have no clue how this will be changed in future o detailed requirements. It's probabile this feature will be wrapped in a template or a diagram-settings-tab. At this time we need this to make it works for some clients templates